TreeTracker/UI/Views/View Trees/ViewTreesViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewTreesViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 10/27/20.
+//  Copyright © 2020 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ViewTreesViewController: UIViewController {
+    var viewModel: ViewTreesViewModel? {
+        didSet {
+            viewModel?.viewDelegate = self
+            title = viewModel?.title
+        }
+    }
+   @IBOutlet weak var viewTrees: UICollectionView!
+   //private let treeMonitoringService: TreeMonitoringService
```

Comment:
Can we remove this commented out line if its not used.

Reply:
I will fix